portal-api/api-tests/v1/gef/facilities.api-test.js

```diff
@@ -366,6 +366,39 @@ describe(baseUrl, () => {
       const { status } = await as(aMaker).put({}).to(`${baseUrl}/doesnotexist`);
       expect(status).toEqual(204);
     });
+
+    it('should update the coverStartDate and coverEndDate in the right format set to midnight', async () => {
+      const { details } = newFacility;
+      const update = {
+        hasBeenIssued: true,
+        name: 'Test',
+        currency: { id: 'GBP' },
```

Comment:
Do we have a constant for currency `GBP`, if not then please ignore. :-)

Reply:
done